docs/GettingStartedDocs/buildandsign.md

```diff
@@ -8,15 +8,15 @@ Also like the Intel SGX SDK, these binaries must be built with a specific set of
 To simplify the process of the specifying the correct build parameters, Open Enclave SDK provides
 a set of `pkg-config` settings that you can use in your build scripts.
 
-There are 8 configurations provided to handle all permutations of the following parameters:
```

Comment:
This document is related to building samples and does not need to modify.

Reply:
Thanks for the clarification, I have reverted the changes to this file.